node/src/reactor/validator/tests.rs

```diff
@@ -235,29 +211,37 @@ async fn run_equivocator_network() {
     keys.push(alice_sk);
 
     let mut chain = TestChain::new_with_keys(&mut rng, keys, stakes);
+    let protocol_config = (&*chain.chainspec).into();
 
     let mut net = chain
         .create_initialized_network(&mut rng)
         .await
         .expect("network initialization failed");
 
+    info!("Waiting for Era 0 to end");
+    net.settle_on(&mut rng, is_in_era(EraId(1)), Duration::from_secs(600))
+        .await;
+
     info!("Waiting for Era 1 to end");
-    net.settle_on(&mut rng, is_in_era(1), Duration::from_secs(600))
+    net.settle_on(&mut rng, is_in_era(EraId(2)), Duration::from_secs(90))
         .await;
 
     info!("Waiting for Era 2 to end");
-    net.settle_on(&mut rng, is_in_era(2), Duration::from_secs(90))
+    net.settle_on(&mut rng, is_in_era(EraId(3)), Duration::from_secs(90))
         .await;
 
     info!("Waiting for Era 3 to end");
-    net.settle_on(&mut rng, is_in_era(3), Duration::from_secs(90))
+    net.settle_on(&mut rng, is_in_era(EraId(4)), Duration::from_secs(90))
         .await;
 
-    info!("Waiting for Era 4 to end");
-    net.settle_on(&mut rng, is_in_era(4), Duration::from_secs(90))
+    let last_era_id = EraId(5);
+    println!("Waiting for Era {} to end", last_era_id.0 - 1);
+    net.settle_on(&mut rng, is_in_era(last_era_id), Duration::from_secs(90))
         .await;
 
-    println!("Waiting for Era 5 to end");
-    net.settle_on(&mut rng, is_in_era(5), Duration::from_secs(90))
-        .await;
+    // Make sure we waited long enough for this test to include unbonding and dropping eras.
+    let oldest_bonded_era_id = consensus::oldest_bonded_era(&protocol_config, last_era_id);
+    let oldest_evidence_era_id =
+        consensus::oldest_bonded_era(&protocol_config, oldest_bonded_era_id);
+    assert!(!oldest_evidence_era_id.is_genesis());
```

Comment:
So you want to make sure that we dropped an era after Genesis?

Reply:
No, I just want to make sure that we drop _genesis_. Which means that the oldest era that is still needed is _not_ genesis.

Reply:
Ah right, b/c `oldest_evidence_era_id` is still kept in `EraSupervisor`'s memory.